### PR TITLE
UI update, dark mode + misc. changes

### PR DESCRIPTION
**Pull request by ItsPepperpot**
:shipit:

### CODE REVIEW

File: MainWindow.xaml [hunk 1]
```diff
@@ -5,56 +5,92 @@
...
+
+            <Label Foreground="{StaticResource ForegroundBlueStandardBrush}" FontFamily="./Fonts/#Poppins" FontWeight="DemiBold" FontSize="12" VerticalAlignment="Bottom" HorizontalAlignment="Left">Install directory</Label>
+            <StackPanel Orientation="Horizontal">
+                <TextBox HorizontalAlignment="Left" FontFamily="./Fonts/#Poppins" Background="{StaticResource BackgroundLightBrush}" Width="260" x:Name="InstallDir" Grid.Column="1" Grid.ColumnSpan="5" Grid.Row="16" FontSize="14" Padding="4" Margin="12,0,12,0" Foreground="{StaticResource ForegroundWhiteStandardBrush}"/>
+                <Button x:Name="Browse" Background="{StaticResource BackgroundLightBrush}" FontSize="12" Foreground="{StaticResource ForegroundWhiteImportantBrush}" MinWidth="40" FontFamily="./Fonts/#Poppins" Click="BrowseFolder">...</Button>
+            </StackPanel>
+
+
+            <TextBlock x:Name="Game" FontFamily="./Fonts/#Poppins" FontSize="1" TextAlignment="Center" VerticalAlignment="Center"></TextBlock>
```
Comment (Suprnova) [inline MainWindow.xaml]: This crunches the Game field and makes it impossible to display under normal behavior, and should be fixed by adjusting the layout of the window.

File: MainWindow.xaml [hunk 2]
```diff
@@ -5,56 +5,92 @@
...
+        </StackPanel>
+        
+        <!-- Developer info -->
+        <TextBlock
+            Foreground="{StaticResource ForegroundWhiteStandardBrush}" 
+            FontFamily="./Fonts/#Poppins"
+            Grid.Column="0"
+            Grid.Row="2"
+            HorizontalAlignment="Center" TextAlignment="Center">
+            Developed by Suprnova<LineBreak/>View on <Hyperlink TextDecorations="None" Foreground="{StaticResource ForegroundPurpleBrush}" NavigateUri="https://github.com/Suprnova/Shell-City-Mod-Loader">GitHub</Hyperlink></TextBlock>
```
Comment (Suprnova) [inline MainWindow.xaml]: do me a favor and add your name here too for ui design bestie

Comment (ItsPepperpot) [reply MainWindow.xaml]: babe 😳😳😳

**Comment by Suprnova** [comment]
thank you pepperpot i love you platonically please bear my children

**Pull request merged**